docs/widgetbook-cloud/integrating-github.mdx

```diff
@@ -0,0 +1,57 @@
+# Integrating GitHub App with Widgetbook Cloud
+
+A GitHub App is a first-class actor on GitHub, meaning it has its own identity separate from users.
+GitHub Apps can perform tasks such as sending messages or opening issues, and they can be installed directly on organizations and user accounts and granted access to specific repositories. They offer granular permissions, enhancing security by limiting access to only what's necessary.
+
+To streamline your workflow and improve your overall efficiency, you can integrate GitHub App with Widgetbook Cloud.
+
+<video width="100%" height="720px" autoplay muted loop>
+  <source src="/assets/integrating-github/github-integration-hd.mp4" type="video/mp4" />
+  Your browser does not support the video tag.
+</video>
+
+Here's a step-by-step guide on how to go about it:
+
+## Step 1: Initiate via your Company Profile
+
+First, navigate to your company's profile on Widgetbook Cloud. This can be done by selecting 'Company' from the navigation rail on your dashboard.
+
+<Image src="/assets/integrating-github/1.png" zoom />
+
+## Step 2: Select your Company
+
+Select the company you want to integrate from the list of available companies with the GitHub App.
+
+<Image src="/assets/integrating-github/2.png" zoom />
+
+## Step 3: Start GitHub App Installation Flow
+
+Now, you're ready to start the GitHub App installation process. This process requires administrative permissions for the repository. If you lack these permissions, ask your repository admin to initiate the process.
+
+<Image src="/assets/integrating-github/3.png" zoom />
```

Comment:
That's not the case. You can still follow the flow but you'll request permission instead and the repo admin has to accept the request of "installing" the GitHub app.